detox/src/devices/DriverRegistry.test.js

```diff
@@ -54,6 +55,14 @@ describe('DriverRegistry', () => {
       expect(EmulatorDriver).toHaveBeenCalledWith(opts);
     });
 
+    it('should resolve "stub" to StubDriver', () => {
+      const StubDriver = require('./drivers/stub/StubDriver');
+      const driver = registry.resolve('stub', opts);
```

Comment:
I don't like this at all. We should not export our internal for-test code to the end users. Why can't you use a direct path to the driver file instead, and put the driver inside `detox/test/integration`? @d4vidi

Reply:
Why - what's the concern? This isn't an external driver per se

Reply:
It is used only one place so far, to test one thing here for us. Why it should be packaged to npm registry at all, e.g.?

Reply:
I'm actually intrigued to see whether the community could find something creative to do with this 😆

Reply:
ok ok I've applied that in my latest commit. Please resolve :)